quiz.css

```diff
@@ -0,0 +1,85 @@
+* {
```

Comment:
**Pro tip:** The * selector in CSS is a very expensive (i.e. memory intensive) operation for the browser.  Always be as specific as you can possibly be.